ladybug_display_schema/visualization.py

```diff
@@ -293,27 +269,76 @@ class GraphicContainer(NoExtraBaseModel):
         'If None, the default units of the data_type will be used.'
     )
 
+
+class VisualizationSet(NoExtraBaseModel):
+    """A visualization where multiple data streams correspond to the same geometry."""
+
+    type: constr(regex='^VisualizationSet$') = 'VisualizationSet'
+
+    geometry: Union[Mesh2D, Mesh3D, Polyface3D, List[GEOMETRY_UNION]] = Field(
+        ...,
+        description='A ladybug-geometry object or list of ladybug-geometry objects that '
+        'is aligned with the values in the input data_sets. If a Mesh or Polyface '
+        'is specified here, it is expected that the number of values match the '
+        'number of faces or the number of vertices. If a list of geometry objects '
+        'is specified (ie. a list of Point3Ds), it is expected that the length '
+        'of this list align with the number of values.'
+    )
+
+    data_sets: List[VisualizationData] = Field(
+        ...,
+        min_items=1,
+        description='An list of VisualizationData objects representing the data sets '
+        'that are associated with the input geometry.'
+    )
+
+    default_data: int = Field(
+        0,
+        description='An integer to denote which of the input data_sets should be '
+        'displayed by default.'
+    )
+
+    context: List[GEOMETRY_UNION] = Field(
+        None,
+        description='An optional list of ladybug-geometry objects that should be '
+        'displayed alongside the input geometry to help give the geometry context. '
+        'Typically, these will display in wireframe around the geometry.'
+    )
+
+    min_point: Point3D = Field(
+        None,
+        description='An optional Point3D object for the minimum of the bounding box '
+        'around the visualization geometry. If None, then the bounding box around the '
+        'geometry will be used to set up the position of legends and titles in 3D '
+        'space (if these objects are represented with 3D geometry).'
+    )
+
+    max_point: Point3D = Field(
+        None,
+        description='An optional Point3D object for the maximum of the bounding box '
+        'around the visualization geometry. If None, then the bounding box around the '
+        'geometry will be used to set up the position of legends and titles in 3D '
+        'space (if these objects are represented with 3D geometry).'
+    )
```

Comment:
If the only usage of these points is for locating the legend should we rename them to legend_position? This will only be used in Ladybug Tools Grasshopper.

Reply:
You're right and I think we can actually do away with them and keep them limited to the Ladybug-Core GraphicContainer. We'll just auto-compute these points if we generate GraphicContainers from the VisualizationSet.